site/gatsby-site/src/components/forms/PreviewImageInputGroup.js

```diff
@@ -57,6 +63,11 @@ export default function PreviewImageInputGroup({
           schema,
         }}
       />
+      {imageLoadedFailed && (
+        <span className="text-sm text-red-600">
```

Comment:
It seems to show up even when nothing has been entered. I would add a check that the field has been touched.

Elsewhere when we have red error text, we also put a red border on the input box.

Reply:
I added the touched. So now the warning is displayed only after an invalid URL is added.

However, I thought about adding the red border to the input but I think it looks too much like an error instead of a warning. 
I think it would make the user think they are required to enter a valid URL.